### ISSUE TITLE
RateLimiterQueue: getTokensRemaining broken for RateLimiterPostgres after key expires

### ISSUE THREAD

**Issue by jhurwitz** [animir/node-rate-limiter-flexible#125]
I believe this should repro the error:
```
const rateLimiter = new RateLimiterPostgres(
  {
    points: 5,
    duration: 1,
    storeClient: <CLIENT>,
    storeType: 'knex',
    tableName: '<TABLE_NAME>',
    keyPrefix: '',
    tableCreated: true,
  },
);
const queue = new RateLimiterQueue(rateLimiter);

await queue.getTokensRemaining('key');
await queue.removeTokens(1, 'key');
await queue.getTokensRemaining('key');
await new Promise((resolve) => setTimeout(resolve, 3000));
await queue.getTokensRemaining('key');
```

The first two calls to `getTokensRemaining` succeed, but the third fails with: `Cannot read property 'remainingPoints' of null`

Based on debugging this, I think the root cause is this: On the third call, the `expire` value stored in postgres is in the past, so the `_get` method of `RateLimiterPostgres` returns null.

**Comment by animir** [comment]
@jhurwitz Hi, thank you for reporting this bug

**Comment by animir** [comment]
Fixed in version `2.3.5`
